README.md

```diff
@@ -94,6 +101,21 @@ observer.join_channel('channel')
 observer.leave_channel('channel')
 ```
 
+### 5. Deinstallation
```

Comment:
I personally like if people provide a way/information on to remove software. Not just how to get it installed. But I get your point. It should be obvious.